pkg/view/router.go

```diff
@@ -401,9 +415,15 @@ func New(
 		}
 
 		form.Name = r.Form.Get("name")
-		t := r.Form.Get("type")
+		form.Type = r.Form.Get("type")
 		form.RequestID = r.Form.Get("request_id")
 
+		// XXX breadchris gorilla/schema fails to parse instance with extra fields
+		// TODO breadchris form should be created with scoped fields so settings can be separated
```

Comment:
Looks like you can just use `IgnoreUnknownKeys()` https://github.com/gorilla/schema/blob/main/decoder_test.go#L868